project/PythonTasks.scala

```diff
@@ -20,15 +21,15 @@ object PythonTasks {
   def buildPythonTask(baseDirectory: File, version: String): Unit = {
     val cwd = workingDirectory(baseDirectory)
     val exitCode = Process(Seq(cwd.getAbsolutePath + "/build." + ext,
-      version, "setup.py"), cwd).!
+      version, pythonExecutable, "setup.py"), cwd).!
```

Comment:
Adding pythonExecutable here makes the variable `PYTHON_EXECUTABLE` not just for test but also for normal build. User can only build a 2.7 wheel for jobserver if he has this has the env variable. This is not specified anywhere. I guess if you can add this in python.md? then the PR seems complete.

Reply:
Done